libsolidity/interface/FileReader.cpp

```diff
@@ -308,6 +302,27 @@ boost::filesystem::path FileReader::normalizeCLIPathForVFS(
 	return normalizedPathNoDotDot;
 }
 
+boost::filesystem::path FileReader::normalizeCLIRootPathForVHS(
+	boost::filesystem::path const& _path,
+	boost::filesystem::path const& _base
```

Comment:
I'd explicitly call it "working directory" for clarity. It will still work correctly if it's not work dir (because only its root path matters) but `absolutePath` will not really be the absolute version of `_path`.
```suggestion
	boost::filesystem::path const& _workDir
```
Or, alternatively, maybe it would be better to simply accept the work dir root path here?
```suggestion
	boost::filesystem::path const& _workDirRootPath
```

Reply:
OK. The original parameter name was as for [`boost::filesystem::absolute`](https://www.boost.org/doc/libs/1_68_0/libs/filesystem/doc/reference.html#absolute)

> Or, alternatively, maybe it would be better to simply accept the work dir root path here?

I thought about it too. But that would require cutting the root in several places on  the caller side.

Maybe we can say that `_workDir` defines the context on normalization, and taking into account just the root is an implementation detail.

But I would not mind `_workDirRootPath = boost::filesystem::current_path().root_path()` either :).

Reply:
> The original parameter name was as for [boost::filesystem::absolute](https://www.boost.org/doc/libs/1_68_0/libs/filesystem/doc/reference.html#absolute)

Ah, ok. For `absolute()` it makes sense. For our normalization we assume work dir though.

> I thought about it too. But that would require cutting the root in several places on the caller side.

It's just a call to `root_path()` so I don't think it's that bad. Having the function do it is more convenient of course but then it's getting more info than it really needs.

In any case, I'm fine with either.